prusa/link/web/main.py

```diff
@@ -51,6 +51,15 @@
     run directly from your printer's internal storage)."""
 
 
+def filter_none(dictionary):
```

Comment:
There is some similar function in SDK, please check it.

Reply:
Used filter_null from SDK